frontend/src/scenes/instance/SpecialMigrations/index.tsx

```diff
@@ -29,20 +36,28 @@ export function SpecialMigrations(): JSX.Element {
         {
             title: '',
             render: function RenderTriggerButton(specialMigration: SpecialMigration): JSX.Element {
+                const status = specialMigration.status
                 return (
-                    <div>
-                        {specialMigration.status === 0 ? (
+                    <Tooltip title={tooltipMessageForStatus[status]}>
+                        {status === 0 ? (
                             <PlayCircleOutlined
-                                className="trigger-migration-button"
+                                className="migration-btn success"
                                 onClick={() => triggerMigration(specialMigration.id)}
                             />
-                        ) : specialMigration.status === 1 ? (
+                        ) : status === 1 ? (
                             <StopOutlined
-                                className="force-stop-migration-button"
+                                className="migration-btn danger"
                                 onClick={() => forceStopMigration(specialMigration.id)}
                             />
-                        ) : null}
-                    </div>
+                        ) : status === 2 ? (
+                            <CheckCircleOutlined className="success" />
+                        ) : (
```

Comment:
for safety we might want to be explicit here about status 3 & 4 so we don't accidentally allow re-runs on whatever 5 is in the future